### PR TITLE
docs: clarify how to login in remote mode

### PR DESCRIPTION
**Pull request by RomainGoussault**

### CODE REVIEW

File: substrafl_examples/get_started/run_mnist_torch.py [hunk 1]
```diff
@@ -44,24 +44,32 @@
...
+# Other backend_types are:
+#
+# - `docker` mode where computations run locally in docker containers
+# - `remote` mode where computations run remotely (you need to have a deployed platform for that)
+#
+# To run in remote mode, use the following syntax:
+#
+# `client_remote = Client(url="MY_BACKEND_URL")`
+#
+# `client_remote.login(username="my-username", password="my-password")`
```
Comment (RomainGoussault) [inline substrafl_examples/get_started/run_mnist_torch.py:63]: I can't manage to have this displayed properly. It's either:

Comment (RomainGoussault) [reply substrafl_examples/get_started/run_mnist_torch.py:63]: image

Comment (RomainGoussault) [reply substrafl_examples/get_started/run_mnist_torch.py:63]: Or 
```
# `client_remote = Client(url="MY_BACKEND_URL")`
#
# `client_remote.login(username="my-username", password="my-password")`
```
image

Comment (RomainGoussault) [reply substrafl_examples/get_started/run_mnist_torch.py:63]: @ThibaultFy @SdgJlbl any idea?

Comment (SdgJlbl) [reply substrafl_examples/get_started/run_mnist_torch.py:63]: And with double backticks? I _think_ it's the way to display code in RsT 🤔

**Review by ThibaultFy — APPROVED** [review]
Thanks :)

**Pull request merged**